src/docs/sdk/cache.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Cache
+sidebar_order: 20
+---
+
+The SDK's caching integration is agnostic to the underlying engine used. In most cases, the SDK will instrument existing cache abstractions in certain frameworks or libraries.
+
+## Span conventions
+
+### Span Operations
+
+| Span OP | Description |
+|:--|:--|
+| `cache.get` | A single item or multiple items are retrieved from the cache |
+| `cache.put` | A single item or multiple items are written to the cache |
+| `cache.remove` | A single item or multiple items are removed from the cache |
+| `cache.flush` | The entire content of the cache is deleted |
+
+### Span Data
+
+| Data Key | Type | Description | Conditions
+|:--|:--|:--|:--|
+| `cache.hit` | bool | cache hit or miss | required only on read-operations (`cache.get`)
+| `cache.key ` | array | the key(s) involved in the operation | required only on operations that have a key
+
+The following data attributes are only to be set if exposed by the instrumented cache abstraction or retrievable with minimal overhead.
+
+| Data Key | Type | Description | Conditions
+|:--|:--|:--|:--|
+| `cache.item_size ` | int | the size of the item/items read/written/deleted in `bytes` | only on operations that have a key
```

Comment:
Retrieving the size might be expensive, hence I would opt to make this optional.

Reply:
Would retrieving the size be a flag set in the sdk config?

Reply:
We could make this an opt-in config on the integration level, in case the used abstraction does not provide this data.

I'm mostly concerned about the need to serialize none scalar values to gather their size.